src/framework/preferences/messages/contrast_fa.json

```diff
@@ -1,5 +1,5 @@
 {
-    "contrast-default": "پیش فرض",
+    "contrast-default": "No Preference",
```

Comment:
The translation should be in Farsi instead of English.

The same comment applies to translation of "No Preferences" in contrast_fr.json, contrast_pt_BR.json.

Reply:
Thanks for catching this. I had forgotten that those files were updated as well. Interesting thing I had missed before is that the localization preferences already used "No Preference" instead of "Default".